_includes/two_image_block.html

```diff
@@ -0,0 +1,10 @@
+<!-- This has a path prefix to make the usage in the compactor process memory blog post more concise -->
+<!-- can be adapted to be more generic if needed -->
```

Comment:
I didn't know you could pass parameters to includes like this. That's neat. But, if we want this include to be more general purpose, you should push the full image URL into the calling code, and omit it here. Can also make the alt text a parameter.